src/test/java/com/kainos/ea/backend/controllers/UserControllerTest.java

```diff
@@ -0,0 +1,123 @@
+package com.kainos.ea.backend.controllers;
+
+import com.kainos.ea.backend.models.User;
+import com.kainos.ea.backend.services.UserService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+public class UserControllerTest {
+
+    @Mock
+    private UserService userService;
+
+    @Mock
+    private BCryptPasswordEncoder passwordEncoder;
+
+    @Test
+    public void when_verifyUser_called_with_valid_user_expect_true() {
```

Comment:
What's the naming convention here? I don't mind as long as it is consistent but I would have assumed it might have been either 
`when_verify_user_called_with_valid_user_expect_true`
or 
`when_verifyUserCalledWithValidUser_expect_true` (when_thingsInCamelCase_expect_correctAnswerInCamelCase)

Reply:
Changed the test names to match with the convention